pkg/utils/helper.go

```diff
@@ -218,6 +284,24 @@ func CreateCRD() error {
 	return nil
 }
 
+// GetResourceList gets a list of all resources of given type in a specified namespace
+func GetResourceList(kubeCmds []string, separator string) ([]string, error) {
+	outputStreams, err := RunCommandOnContainerWithOutputStreams("kubectl", kubeCmds...)
+
+	if err != nil {
+		return nil, err
+	}
+
+	resourceList := outputStreams.Stdout
+	// If the resource is not found within the cluster, then log a message and do not return any resources.
+	if len(resourceList) == 0 {
+		err := fmt.Errorf("No '%s' resource found in the cluster for given kubectl command", kubeCmds[1])

```

Comment:
`return nil, fmt.Errorf("No '%s' resource found in the cluster for given kubectl command", kubeCmds[1])`